hooks/hook-install.sh

```diff
@@ -1,5 +1,8 @@
+#!/usr/bin/env bash
+
 sudo apt-get install mongodb
 sudo apt-get install python-dev
+sudo apt-get install python-pil
 pip install bottle gunicorn pymongo requests
-nohup gunicorn -w 2 app:hook-example -b 0.0.0.0 > /tmp/gunicorn.log &
+nohup gunicorn -w 2 -b 0.0.0.0:80 hook-example:app > /tmp/gunicorn.log &
```

Comment:
I think you said 8080 as port elsewhere in the doc so check that port is same throughout.